updatecli/updatecli.d/updatebuildbase.yaml

```diff
@@ -0,0 +1,68 @@
+---
+name: "Update build base version" 
+
+sources:
+  gomod:
+    name: Get latest Golang version based on go.mod
+    kind: file
+    spec:
+      file: https://raw.githubusercontent.com/kubernetes-sigs/cri-tools/master/go.mod
```

Comment:
Query the go version from the `master` branch instead of the tag defined by the [image-build](https://github.com/rancher/image-build-crictl/blob/71d73c439cf6f442247f8f94dc5ffc46d1a54fec/Dockerfile#L15)?

```
$ curl https://raw.githubusercontent.com/kubernetes-sigs/cri-tools/v1.26.1/go.mod 2>/dev/null | egrep 'go ([0-9]+\.[0-9]+)'
go 1.19
$ curl https://raw.githubusercontent.com/kubernetes-sigs/cri-tools/master/go.mod 2>/dev/null | egrep 'go ([0-9]+\.[0-9]+)'
go 1.22.0
```

Reply:
I thought about that but if you check the releases, we are not really updating that tag and releasing new versions that overwrite that tag. I thought using the master would be ok since we want to be always in sync with upstream k8s releases